sw/host/opentitanlib/src/app/mod.rs

```diff
@@ -720,29 +724,35 @@ impl TransportWrapper {
     /// Returns a SPI [`Target`] implementation.
     pub fn spi(&self, name: &str) -> Result<Rc<dyn Target>> {
         let name = name.to_uppercase();
+        let mut spi_logical_map = self.spi_logical_map.borrow_mut();
```

Comment:
Do these fields get cleared on a `transport init` call? They should

Reply:
I had not thought of that, but you are right.  If someone have manually changed the speed of the port while using e.g. the "TPM" alias, we do not want that manual speed setting to be used when the "TPM" alias is used after next `opentitantool transport init`.   I have added code to clear these cache maps in `apply_default_configuration()`